### PR TITLE
global: addition of version field

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
<a href="https://i.imgur.com/UhzqLUC.png">
  
</a>

**Comment by krzysztof** [comment]
@slint can you take a look?

File: zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json [hunk 1]
```diff
@@ -256,6 +256,38 @@
         ]
       },
       {
+        "type": "text",
+        "key": "version",
+        "title": "Version tag",
+        "description": "Optional. Mostly relevant for software and dataset uploads. Any string will be accepted, but semanticly-versioned tag is recommended.<br />See <a href='http://semver.org'><i class='fa fa-external-link'></i> semver.org</a> for more information on semantic versioning.",
```
Comment (slint) [inline zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json]: `s/semanticly/semantically`

**Comment by krzysztof** [comment]
Integrated in another PR

**Pull request closed without merging**